README.md

```diff
@@ -13,6 +13,16 @@ or connect it to the other parts of the Biomage Single Cell Platform.
 
 We hihgly recommend using VSCode for local development. Make sure you also have `npm` and `docker` installed.
 
+You will also need to have `aws-cli` installed and cofigured. See documentation in here: https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html
```

Comment:
* `cofigured` :-)
* I would keep the explicit reference to running `aws configure` that you added before
* I would remove the reference that you found in the worker section
* lets go for status quo regarding brew :-)

Reply:
👍

Reply:
Thanks! Good point regarding MacOS.